releasenotes/notes/30200.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: istioctl
+issue:
+  - 30200
+releaseNotes:
+  - |
+    **Fixed** wasteful re-runs of analyzer by making the analysis run interval configurable
```

Comment:
Well its not really fixed.. if you configure it to be higher then you just get stale data.

Seems like ideally we would have event based analysis. Not suggesting we do that since its probably quite challenging...

Reply:
@howardjohn Thanks for reviewing this PR so fast, the idea here is to provide an option to allow admin to check things according to their own need. Mitch and I discussed this a bit, event based idea seems require a more code and  watching a lot of resources can also add load to a cluster, the pros and cons certainly debatable.  As you've noted it may ask for a lot of code.  The timer based implementation actually is simpler, it may be the reason why it was implemented currently, with this PR, an admin can manipulate this parameter based on their understanding of their own cluster, they can observe their cluster by setting different values over a period of time until their cluster is relatively stable.

Reply:
Let's change the `kind` of this release-note to `feature` to better reflect that we are not closing a bug here.

Reply:
Probably also change the word **Fixed** above.  You can look through the other release notes to see what new features usually look like...